src/renderer/wddmcon/main.cxx

```diff
@@ -16,7 +16,7 @@ HINSTANCE g_hInstance;
 #define REGSTR_VALUE_DISPLAY_INIT_DELAY  L"DisplayInitDelay"
 #define REGSTR_VALUE_FONT_SIZE  L"FontSize"
 
-#define SafeRelease(p) if (p) { (p)->Release(); (p) = NULL;}
+#define SafeRelease(p) if (p) { (p)->Release(); (p) = nullptr;}

```

Comment:
FYI I deliberately didn't reformat this file, because it would rewrite pretty much every other line and I felt like that's out of the scope of just making it work.